.gitignore

```diff
@@ -5,6 +5,3 @@ content/data/*.db
 content/logs/
 config.production.json
 content/data/ghost-local.db
-
-content/themes/*
```

Comment:
The `content/themes/` need to be excluded because they're meant to be copied over from `node_modules`, NOT hardcoded in the repo.

Revert this change.

Reply:
@JaneJeon I don't think this should be a problem. See my points on why pushing your themes to this repository might be a better idea. Also, since the included themes are copied over to content/themes after the install step (which happens only on heroku), we should not have to worry about getting them there in `content/themes` in the source code

Reply:
Of *course* all the themes belong in `content/themes`, as specified by Ghost CMS.

The question is whether you want to check the raw theme code into your codebase, and that's really just a hacky AF solution. Come up with a better way (that doesn't involve checking in individual themes) or remove this change entirely.